kupyna/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Please use the LICENSE-MIT and LICENSE-APACHE files. You can copy them from another hash crate.